free-2.md

```diff
@@ -58,12 +110,23 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I think that if someone were to work on this project, perhaps only 20% of the work is spent directly engaging in the language aspects of this project (e.g., making language design decisions). This DSL is very abstracted, and demands very little of the user’s ability to code in traditional programming languages. The user is only interacting with an excel spreadsheet. 
+The bulk of the work would be spent on the "systems" aspects of the project (e.g., implementing a complicated semantics that doesn't require a lot of language design), because of the need for ----  
+
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+I think the greatest challenge is to communicate information to an existing musical notation software. It’s very easy for me to see the excel spreadsheet and write sheet music, but I have NO IDEA how to automate that process. 
+However, I know this process would involve teaching the computer about different types of chords, the structure of different intervals that compose a chord, so that I don’t have to explicitly teach the program every possible chord. 
+Example: I teach my program about the structure of major chords, minor chords, diminished chords, major 7th chords, and dominant 7th chords. Each chord is comprised of and defined by a particular set of intervals between various notes. I would also teach the program about the different inversions that a chord can be presented in. The most challenging part of the project is, the program will need to learn what it means for a certain chord to have a given root or key signature, so that it would be able to generate all the possible chords given a basic set of rules. 
+
```

Comment:
You mentioned that you wanted the output of your program to be a MuseScore file.  It seems that MuseScore has an API which would be useful for generating these files.  I am not sure how difficult it would be to generate and edit files with this API, but it would be worth investigating.  Alternatively, there exist other DSLs that can generate sheet music much more broadly, such as LilyPond (I mentioned it in your previous file).  I think the output of your program would be different, but I'm sure of the benefits of the different file types.  Also, I think you could write a python script that reads an Excel file.  You seem to have a pretty good grasp about how to "train" the computer to read different types of chords, so I think this DSL as a whole is feasible.